x-pack/plugins/observability/common/guided_onboarding/kubernetes_guide_config.tsx

```diff
@@ -28,30 +28,26 @@ export const kubernetesGuideConfig: GuideConfig = {
     {
       id: 'add_data',
       title: i18n.translate('xpack.observability.guideConfig.addDataStep.title', {
-        defaultMessage: 'Add and verify your data',
+        defaultMessage: 'Add data',
       }),
       integration: 'kubernetes',
-      descriptionList: [
-        {
-          descriptionText: i18n.translate(
-            'xpack.observability.guideConfig.addDataStep.descriptionList.item1.descriptionText',
-            {
-              defaultMessage: 'Deploy kube-state-metrics service to your Kubernetes.',
-            }
-          ),
-          linkText: i18n.translate(
-            'xpack.observability.guideConfig.addDataStep.descriptionList.item1.linkText',
-            {
-              defaultMessage: 'Learn more',
-            }
-          ),
-          linkUrl: 'https://github.com/kubernetes/kube-state-metrics',
-          isLinkExternal: true,
-        },
-        i18n.translate('xpack.observability.guideConfig.addDataStep.descriptionList.item2', {
-          defaultMessage: 'Add the Elastic Kubernetes integration.',
-        }),
-      ],
+      description: {
+        descriptionText: i18n.translate(
+          'xpack.observability.guideConfig.addDataStep.description.descriptionText',
+          {
+            defaultMessage:
+              'Install an Elastic Agent to get your Kubernetes data flowing. Optionally, deploy kube-state-metrics to your Kubernetes for more comprehensive metrics coverage.',
```

Comment:
```suggestion
              'Install Elastic Agent to get your Kubernetes data flowing. Deploy kube-state-metrics to your Kubernetes for more comprehensive metrics coverage.',
```

Reply:
@mlunadia Can you please approve this text?

Reply:
'To get your Kubernetes data flowing, install Elastic Agent in the Kubernetes cluster you want to monitor. Once Elastic Agent is deployed, you can optionally add [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) for a more comprehensive metrics coverage.'

Reply:
@mlunadia Because of the technical constrains, we can't put the link `https://github.com/kubernetes/kube-state-metrics` in text. But it's being appended as [Learn more](https://github.com/kubernetes/kube-state-metrics) at the end of the description.

Reply:
The text that @mlunadia wrote LGTM.  In this [Elastic doc](https://docs.elastic.co/en/integrations/kubernetes/kube-state-metrics), kube-state-metrics is in code font, so we should do the same here.  

Can you post a screenshot with the updated text?

Reply:
Should be kube-state-metrics, at a glance I can see that document quoting kube_state_metrics as a field but as a noun therefore we can update it for kube-state-metrics as used in https://github.com/kubernetes/kube-state-metrics

Reply:
Here is a screenshot of the latest suggestion 
<img width="461" alt="Screenshot 2023-02-07 at 15 38 43" src="https://user-images.githubusercontent.com/6585477/217275346-03007015-bc6c-4033-9a64-65da8df299b7.png">

@gchaps Unfortunately, the same as with the link, we can't use code font here.